resources/assets/ts/DarkMode.ts

```diff
@@ -0,0 +1,80 @@
+/*!
```

Comment:
where is this file from?
do we have to mention this in the readme?

Reply:
From here: https://getbootstrap.com/docs/5.3/customize/color-modes/#javascript